lessons/Projects/project-infographic/langs/en-us/index.adoc

```diff
@@ -0,0 +1,95 @@
+|===
+| Lesson Goals
+| Students will be able to...
+
+* think critically and creatively about ratio relationships
+* represent ratio relationships using either repeated images, scaled images, or area models
+
+| Student-facing Lesson Goals
+|
+
+* Let's make an infographic on a topic of interest!
+
+| Materials
+|[.materials-links]
+
+@material-links
+
+| Supplemental Materials
+|[.materials-links]
+@opt-material-links
+
+| Preparation
+|
+* Think about what data will be the basis of your students' infographics. We offer four different options in the "Investigate" section of this lesson. Some require teacher preparation.
+* Students may create their final product with construction paper, markers and art supplies - or using a slide deck / word processor. It's up to you.
+* This project is designed to span multiple days. Consider what work you want students to complete at home and at school.
+
+|===
+
+== Make an Infographic @duration{flexible}
+
+=== Overview
+
+Students get curious about a variety of provided infographics, then complete research and plan to create their own - either electronically, or with paper and art supplies.
+
+=== Launch
+
+A good *infographic* is worth a thousand words!
+
+Infographics are graphic visual representations of data, information, or knowledge that present information quickly and clearly. You probably see infographics all around you - in advertisements, political campaigns, your textbooks, on the news... they're everywhere, because they are so impactful.
+
+@teacher{Explain to students that they are going to create their own infographics. To launch the project, they will look at and consider a variety of infographics. As you peruse the infographics in the @online-exercise{https://docs.google.com/presentation/d/14D98yI7XPzZ-e8PDJrP0iXrvVNxhsFjm_m-1goKSRuw/edit?usp=sharing, "Sample Infographics"} slide deck together, discuss as a class what students observe.
+
+Before you dive in, we recommend printing and distributing the student-facing @handout{rubric-infographic.adoc} to help students understand the scope of the project and your expectations at the outset. Teachers are welcome and encouraged to edit and adapt the rubric for their unique classroom context.
+
+As you move through the slide deck, discuss the questions below.}
+
+[.lesson-instruction]
+- What do you Notice? What do you Wonder?
+- Are the words on the infographic clear and impactful? Why or why not?
+- Are the images in the infographic impactful? Why or why not?
+- Is the infographic accurate (e.g., every element is in the same proportion)?
+- Why do you think each author chose to use a certain strategy to represent their ratio statement (e.g., illustrate ratios using repeated images)?
+
+Emphasize to students that a tricky thing about making infographics with images of people is that not all images accurately represent the diversity of the communities described by the statistics. We encourage the use of silhouetted images in infographics.
```

Comment:
1. This could be student-facing: by simply removing the first four words. "A tricky thing about..."
2. If it's _not_ going to be student-facing, it should use `@teacher`
3. If you're not going to cut the first four words, at least cut words 2+3 ;-)